Anima/diffusion/mcm_estimator/animaMCMEstimatorImageFilter.h

```diff
@@ -437,9 +415,15 @@ class MCMEstimatorImageFilter :
     bool m_UseCommonConcentrations;
     bool m_UseCommonExtraAxonalFractions;
 
-    double m_AxialDiffusivityFixedValue;
-    double m_RadialDiffusivity1FixedValue;
-    double m_RadialDiffusivity2FixedValue;
+    double m_AxialDiffusivityValue;
+    double m_RadialDiffusivityValue;
+    double m_IRWDiffusivityValue;
+    double m_StaniszDiffusivityValue;
+
+    // To do ? Radial diffusivities 1 and 2 are useless for now since there is coarse grid initialization for tensors
```

Comment:
Not sure about that. In the case of the `Tensor` compartment, you still have two radial diffusivities to estimate. Or maybe I am missing some point here?

Reply:
See other comment, we do a grid search over the bounds for radial diffs 1 and 2 now in tensor mode